helpers/helpers.v2.1.d/utils

```diff
@@ -228,9 +228,14 @@ _ynh_apply_default_permissions() {
     # App files can have files of their own
     if ynh_system_user_exists --username="$app"; then
         # If this is a file in $install_dir or $data_dir : it should be owned and read+writable by $app only
-        if [ -f "$target" ] && (is_in_dir "$target" "${install_dir:-}" || is_in_dir "$target" "${data_dir:-}" || is_in_dir "$target" "/etc/$app"); then
-            chmod 600 "$target"
-            chown "$app:$app" "$target"
+        if (is_in_dir "$target" "${install_dir:-}" || is_in_dir "$target" "${data_dir:-}" || is_in_dir "$target" "/etc/$app"); then
+            if [ -d "$target" ]; then
+              chmod -R o+rwX "$target"
```

Comment:
Seems a security hole. If we do `ynh_setup_source` or `ynh_setup_source "$install_dir"` we apply here a o+rwX permission by default, but we should not allowed other users to access this dir (by default) i guess.

In more adding a -R options onto data_dir could be looooong (if there is a lot of file), so this should be done only one time by yunohost install or upgrade...

I suggest to study all chown, chmod apps to understand what should be done by default and how give advice in the right way to packagers if needed.

Reply:
It was supposed to be `u` not `o` x_x